dockerCompose/dockerComposeCompletionItemProvider.ts

```diff
@@ -55,12 +59,14 @@ export class DockerComposeCompletionItemProvider implements CompletionItemProvid
         return Promise.resolve([]);
     }
 
-    private suggestKeys(word: string): CompletionItem[] {
-        return Object.keys(DOCKER_COMPOSE_KEY_INFO).map(ruleName => {
+    private suggestKeys(word: string, useV2Schema: boolean = false): CompletionItem[] {
```

Comment:
I opened a [PR](https://github.com/lostintangent/vscode-docker/pull/1) to address this, but I'll let you decide if you want it or not!